cmd/abapEnvironmentPullGitRepo.go

```diff
@@ -80,7 +68,7 @@ func runAbapEnvironmentPullGitRepo(options *abapEnvironmentPullGitRepoOptions, t
 	}
 
 	if err == nil {
-		err = pullRepositories(repositories, connectionDetails, client, pollIntervall)
+		err = pullRepositories(repositories, connectionDetails, client, pollIntervall, options.IgnoreCommit)
```

Comment:
Just an idea: You could check if `options.IgnoreCommit` is true and assign empty strings for the repositories `CommitID` values with a helper method. This would also reduce the function parameter amount.

Reply:
Initially, I had the performance in mind. If this ingoreCommit check is done elsewhere, we have to introduce another loop. But maybe readability is more important here. I will consider it